apps/balancer-tools/src/app/poolsimulator/(components)/Menu.tsx

```diff
@@ -232,6 +231,102 @@ function SearchPoolFormWithDataForm({
   );
 }
 
+export function AnalysisMenu() {
+  const {
+    initialData,
+    setInitialData,
+    customData,
+    setCustomData,
+    setAnalysisTokenByIndex,
+    analysisToken,
+    setCurrentTabTokenByIndex,
+  } = usePoolSimulator();
+
+  const [tabValue, setTabValue] = useState<PoolSimulatorFormTabs>(
+    PoolSimulatorFormTabs.InitialData
+  );
+  useEffect(() => {
+    setAnalysisTokenByIndex(0);
+    setCurrentTabTokenByIndex(1);
+  }, []);
+
+  const indexCurrentTabToken = initialData?.tokens.findIndex(
+    ({ symbol }) => symbol.toLowerCase() !== analysisToken.symbol.toLowerCase()
+  );
+
+  return (
+    <>
+      <Sidebar.Header name="Simulation menu" />
+      <Sidebar.Content>
+        <div className="flex flex-col gap-4">
+          <div className="flex flex-col">
+            <span className="mb-2 block text-sm text-slate12">
+              Analysis Token
+            </span>
+            <Select
+              onValueChange={(i) => {
+                if (indexCurrentTabToken === Number(i)) {
+                  setCurrentTabTokenByIndex(
+                    initialData?.tokens.findIndex(
+                      (_, index) => index !== Number(i)
+                    )
+                  );
+                }
+                setAnalysisTokenByIndex(Number(i));
+              }}
+              defaultValue={"0"}
+            >
+              {initialData?.tokens.map(({ symbol }, index) => (
+                <SelectItem key={symbol} value={index.toString()}>
+                  {symbol}
+                </SelectItem>
+              ))}
+            </Select>
+          </div>
```

Comment:
This is just a reiteration of the former component with the form underneath it right? Couldn't we just pass the select as a prop to `IndexMenu` and render it there when we need it to avoid repeating all this code?